src/Config.php

```diff
@@ -371,6 +371,12 @@ protected function parseGeneral()
             $general['branding']['provided_by']
         );
 
+        // Add some common 'development' hosts to the `debug_local_domains` setting:
+        $general['debug_local_domains'] = array_unique(array_merge(
+            (array) $general['debug_local_domains'],
+            ['.dev', 'dev.', 'devel.', 'development.', 'test.', '.test', 'new.', '.new', 'localhost', '.local', 'local.']
+        ));
+
```

Comment:
Doesn't matter. In the off chance your domain is `locahost.org`, you just won't see the notice, no harm done.